app/config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
 
   // TODO: LOGIN_ENABLED Remove line below when login is enabled by default
   // Boolean to indicate if we show the login/logout component
-  loginEnabled: process.env.LOGIN_ENABLED || 'false',
+  loginEnabled: process.env.LOGIN_ENABLED || 'true',
```

Comment:
Not sure, so just checking - do we want login to be enabled by default?

Reply:
👍  I'll turn it back off